content/api/api-metrics.md

```diff
@@ -7,112 +7,685 @@ author: Lawrence Lane
+
+
+### Response Body
+
+The following response body returns a list of all metricFQNs associated to the values submitted.
+```
+{
+  "fqns": [
+    "aws.applicationelb.activeconnectioncount",
+    "aws.applicationelb.clienttlsnegotiationerrorcount",
+    "aws.applicationelb.consumedlcus",
+    "aws.applicationelb.forwardedinvalidheaderrequestcount",
+    "aws.applicationelb.http_redirect_count",
+    "aws.applicationelb.httpcode_elb_3xx_count",
+    "aws.applicationelb.httpcode_elb_4xx_count",
+    "aws.applicationelb.httpcode_target_2xx_count",
+    "aws.applicationelb.httpcode_target_3xx_count",
+    "aws.applicationelb.httpcode_target_4xx_count",
+    "aws.applicationelb.httpcode_target_5xx_count",
+    "aws.applicationelb.newconnectioncount",
+    "aws.applicationelb.processedbytes",
+    "aws.applicationelb.requestcount",
+    "aws.applicationelb.requestcountpertarget",
+    "aws.applicationelb.targetresponsetime",
+    "netuitive.aws.alb.httpcodeelb4xxerrorpercent",
+    "netuitive.aws.alb.httpcodeelb5xxerrorpercent",
+    "netuitive.aws.alb.httpcodeelberrorpercent",
+    "netuitive.aws.alb.requestspersecond",
+    "netuitive.aws.alb.totalelbhttperrors",
+    "netuitive.aws.alb.totaltargethttperrors",
+    "netuitive.aws.alb.unhealthyhostpercent",
+    "netuitive.metrics.collected.percent"
+  ]
+}
+```
+
+{{% /expand %}}
+
+---
+
+
+## GET from /metrics/statistics
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/metrics/getMetricStatisticsUsingGET" theme="info" >}} GET {{< /button >}} This endpoint is deprecated.   
+
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|------------|----------------|---------------|-----------------------------------------------------------------------------------------|
+| fqn | query | string | The fully qualified name for a metric. |
+| duration | query | string | Gives CloudWisdom an ISO 8601-formatted duration time frame to retrieve data. The duration ends at the current time and begins anytime in the past two weeks. The duration parameter will take precedence over startTime and endTime if all attributes are included in your request. |
+| startTime | query | date-time | Start time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| endTime | query | date-time | End time for the query. YYYY-MM-DDT00:001Z format (must include time zone). |
+| rollup | query | Array[string] | Determines frequency of metric sampling. Options are: ZERO, PT5M, PT1H, PT24H. |
+| showValues | query | boolean | Provides the values for the metric statistics (if set to true) or only displays zeroes for the statistics (if set to false). |
+
+
+### Request URL
+
+```
+https://app.metricly.com/metrics/statistics?fqn=aws.applicationelb.httpcode_elb_4xx_count&startTime=2020-01-30T01%3A00%3A00-05%3A00&endTime=2020-01-31T01%3A00%3A00-05%3A00&rollup=PT5M&showValues=false
```

Comment:
Use copy-able code span here too.

URL should be https://app.metricly.com/metrics/statistics?fqn={fqn}&startTime={startTime}&endTime={endTime}&rollup={rollup}&showValues={showValues}